spdx-utils/src/main/kotlin/SpdxLicenseIdMapping.kt

```diff
@@ -49,15 +49,15 @@ object SpdxLicenseIdMapping {
             "EDL-1.0" to BSD_3_CLAUSE,
             "EPL" to EPL_1_0,
             "FreeBSD" to BSD_2_CLAUSE_FREEBSD,
-            "GPL" to GPL_2_0,
-            "GPL-2" to GPL_2_0,
-            "GPL2" to GPL_2_0,
-            "GPLv2" to GPL_2_0,
-            "GPLv3" to GPL_3_0,
-            "LGPL" to LGPL_2_1,
-            "LGPL-3" to LGPL_3_0,
-            "LGPL3" to LGPL_3_0,
-            "LGPLv3" to LGPL_3_0,
+            "GPL" to GPL_2_0_ONLY,
```

Comment:
Commit message: Maybe say "Do not map to deprecated SPDX expressions" to empasize that the input (i.e. the key in the map) may very well still contain a deprecated SPDX license id string.

Reply:
Done.